docs/local_resource.md

```diff
@@ -11,76 +11,27 @@ on signal from the user. For your resource `MyGreatService`, when one of its fil
+## Specifying dependencies
+The `deps` argument allows you to specify file dependencies for your local resource---either as a string (filepath) or a list of strings (list of filepaths).
 
-(Of course, the command you run doesn't have to be on that (only) affects your
-local filesystem. It might, for instance, be a script that is _invoked locally_
-but runs against your k8s cluster.)
+When Tilt detects a change to any of a resource's `deps`, the resource will execute (unless the resource is in `TRIGGER_MODE_MANUAL`, in which case the resource will not execute, but you'll see the "pending changes" indicator next to your resource in the sidebar).
 
-## Local Resource + Live Update = <3
-
-Local Resource and Live Update are two great tastes that taste great together.
-A useful pattern is to have a local resource which creates an artifact
-that is then picked up by a Live Update and synced to a running container.
-For example, here's how you might use this combination to generate protobufs:
-```python
-# whenever `helloworld.proto` changes, generate Python protobufs
-local_resource('proto',
-    cmd='protoc --proto_path=src --python_out=build/gen',
-    deps=['helloworld.proto']
-)
-
-# define your hello-world resource: k8s yaml + a Docker build
-k8s_yaml('hello-world.yaml')
-docker_build('hello-world', '.',
-    ignore='helloworld.proto',  # don't step on the local resource's toes
-    live_update=[ sync('.', '/app') ]
-)
-```
+`deps` are optional. By default, a local resource without `deps` runs once on `tilt up` (and any time you change its definition in your `Tiltfile`). You can always manually trigger a local resource (or any resource) with the "force update" button:
 
-Note that the `docker_build` call specifies `ignore='helloworld.proto'`. This is
-because we DON'T want an edit to that proto file to _directly_ kick off an update to
-our Docker image (in this case, a Live Update). Rather, an edit to `helloworld.proto`
-triggers the local resource to generate protobufs; when these protobufs appear on disk,
-they register as file changes and trigger a Live Update to the `hello-world` resource,
-i.e. they get `sync`'d to the container where `hello-world` is running.
-
-To see this pattern in action, check out [this example repo](https://github.com/windmilleng/local_resource_example).
-
-# Run occasional workflows (locally or against your cluster)
+!["force update" button](assets/img/force-update-button.png)
 
-There's another class of workflow that local resource can help with; commands that
-are a part of your development flow, but that you run only occasionally. This might
-be a task like "ensure GKE user 'foo' exists" or "refresh my credentials" or even
-"blow away my database".
+You might use this pattern for, e.g. a local resource that refreshes tokens/credentials needed by your app; you want to do it once on `tilt up`, and every now and then as needed, but not in response to any particular file changes.
```

Comment:
```suggestion
You might use this pattern to, e.g., define a local resource that refreshes tokens/credentials needed by your app - you want to do it once on `tilt up`, and every now and then as needed, but not in response to any particular file changes.
```

1. "for for example"
2. maybe nitty, but I'm not entirely sure the semicolon is wrong, but it feels better to me as not-a-semicolon